CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: Bug that may contextBroker to crash when passing inappropriate parameter in query context payload (#3055)

```

Comment:
Typo: may contextBroker -> may cause contextBroker

Sorry, my fault, as author of the original suggested text :$

Reply:
Fixed in 2182d329780232c2d7740582653132c4c5e8b6ec